app/models/draft_vaccination_record.rb

```diff
@@ -129,6 +131,10 @@ def already_had?
   # So that a form error matches to a field in this model
   alias_method :administered, :administered?
 
+  def protocol
+    super || :pgd
```

Comment:
Do we need the `super` here? I can't see which class it would be getting this method from?

Reply:
> Do we need the super here?

On second thoughts I don't think so.